src/components/Icon/index.js

```diff
@@ -18,22 +21,53 @@ const propTypes = {
 
     /** Is small icon */
     small: PropTypes.bool,
+
+    /** Is small icon */
+    inline: PropTypes.bool,
 };
 
 const defaultProps = {
     width: variables.iconSizeNormal,
     height: variables.iconSizeNormal,
     fill: themeColors.icon,
     small: false,
+    inline: false,
 };
 
+const inlineTop = Platform.OS === 'ios' ? 1 : 2;
+
 // We must use a class component to create an animatable component with the Animated API
 // eslint-disable-next-line react/prefer-stateless-function
 class Icon extends PureComponent {
     render() {
+        const inline = this.props.inline;
         const width = this.props.small ? variables.iconSizeSmall : this.props.width;
         const height = this.props.small ? variables.iconSizeSmall : this.props.height;
         const IconToRender = this.props.src;
+
+        if (inline) {
+            return (
+                <View style={{
+                    width, height, overflow: 'visible', backgroundColor: 'transparent',
+                }}
+                >
+                    <View style={{
+                        width,
+                        height,
+                        position: 'absolute',
+                        top: inlineTop,
+                    }}
```

Comment:
I am sure you didn't look at my earlier comment. Please go through each section of the style guide. **[Inline styles are forbidden](https://github.com/Expensify/App/blob/main/STYLING.md#inline-styles)**